install/helm/gloo/templates/18-settings.yaml

```diff
@@ -39,6 +39,7 @@ spec:
 {{- end }}
     disableKubernetesDestinations: {{ .Values.settings.disableKubernetesDestinations | default false }}
     disableProxyGarbageCollection: {{ .Values.settings.disableProxyGarbageCollection | default false }}
+    enableAutoWebsocketTransformationPassthrough: {{ .Values.settings.enableAutoWebsocketTransformationPassthrough | default false }}
```

Comment:
nit: Don't need to hold this up as others do this too but for future reference you don't need the default in the helm template. its already in the value file and so this will never be used. In general, I would recommend completely avoiding in-line defaults as it just moves information away from the users as they can easily view default values as documentation. You also have this anyways so this has no impact to leave it.